x/fungible/keeper/evm.go

```diff
@@ -789,6 +788,10 @@ func (k Keeper) CallEVMWithData(
 		}
 
 		if !noEthereumTxEvent {
```

Comment:
Can we add some comments as to what this flag refers to 
As in when to pass valid and when false. 

Might make more sense to add it to the top of the function though .